### PR TITLE
Python wizard

### PR DESCRIPTION
**Pull request by nids2001**
Adds a python wizard to the main help menu.

### CODE REVIEW

**Comment by dcwhite** [comment]
Related issue: #250. @jessdtate @nids2001 should we merge this initial part now, then continue with a part 2 issue? Or continue adding commits here and get it all done with one merge later.

**Comment by dcwhite** [comment]
I tested out the latest and it's really cool. I can think of a lot of little things to streamline it. Let's all try to meet this week @jessdtate @nids2001

**Comment by dcwhite** [comment]
@jessdtate It needs just a little clean up at this point, plus probably one more page at the end as a conclusion page. I feel like we should merge it now then make a new issue for the rest.

File: src/Interface/Application/MainWindowCollaborators.cc [hunk 1]
```diff
@@ -310,6 +313,501 @@ QWizardPage* NewUserWizard::createOtherSettingsPage()
...
+
+QWizardPage* PythonWizard::createLoadNetworkPage()
+{
+  auto page = new PythonWizardCodePage;
+  page->setTitle("Load and Execute a Network");
+  page->infoText->setText("Use the command `scirun_load_network()` to load a saved network from sample networks provided"
+    "\n\n"
+    "Execute network with `scirun_execute_all()` command");
+
+  page->codeEdit->setPlainText("scirun_load_network(\"/Users/nidhipatel/myclone/"
```
Comment (jessdtate) [inline src/Interface/Application/MainWindowCollaborators.cc]: this is hard coded to your path.  Can we load it from the scirundata path instead?  or maybe the current path?  You can get environment variables in python with `os.environ['SCIRUNDATADIR']`

File: src/Interface/Application/MainWindowCollaborators.cc [hunk 2]
```diff
@@ -310,6 +313,519 @@ QWizardPage* NewUserWizard::createOtherSettingsPage()
...
+
+  connect(page->sendButton, &QPushButton::clicked, [this, page](){displayPython_(page->codeEdit->toPlainText());});
+
+  return page;
+}
+
+QWizardPage* PythonWizard::createAddIntPyPage()
+{
+  auto page = new QWizardPage;
+  page->setSizePolicy(QSizePolicy::QSizePolicy(QSizePolicy::Fixed, QSizePolicy::Fixed));
```
Comment (dcwhite) [inline src/Interface/Application/MainWindowCollaborators.cc]:
```suggestion
  page->setSizePolicy(QSizePolicy::Fixed, QSizePolicy::Fixed);
```

Comment (dcwhite) [reply src/Interface/Application/MainWindowCollaborators.cc]: Fix compilation

**Comment by dcwhite** [comment]
@nids2001 This needs a compile fix in order to merge

**Comment by dcwhite** [comment]
Thanks

**Comment by dcwhite** [comment]
@jessdtate I'll test the windows build tomorrow--let's try to get this merged this week

**Pull request merged**